### DISCUSSION TITLE
Support for Prepared Statements?

### DISCUSSION THREAD

**Discussion by kevinelliott** [dart-backend/angel#155] [Q&A]
Would it be possible to add support for prepared statements? This has the potential to improve performance with PostgreSQL quite significantly for apps that have significant load.

**Comment by dukefirehawk** [comment]
I don't think ORM is using prepared statement at the moment. Agree that prepared statement can improve the performance. Will take a closer took at the query generator to find a way to support it along with the other changes and fixes to the ORM.

**Reply by kevinelliott** [reply]
That would be amazing, thanks @dukefirehawk !

**Comment by kevinelliott** [comment]
Now that the push to support dart 3.6 is done, what do you think about adding the support for Postgres prepared statements? This would be amazing.

**Comment by dukefirehawk** [comment]
Prepare statement and transaction will be getting the attention soon as part of the overall performance review since issues related to MySQL and code generation have been resolved.
